hash_practice/exercises.py

```diff
@@ -2,18 +2,49 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n log m)
+        Space Complexity: O(n)
     """

```

Comment:
👍   I would say either O(n) for time complexity if words are of limited length or O(n * m log m) if the words can be of arbitrary length because you're processing n words and sort each word of potentially m letters.